install.sh

```diff
@@ -3342,23 +3395,27 @@ execute_installation() {
     echo # For visual separation
 
     # Automatic updates
-    if setup_automatic_update_checks; then
+    if ! is_update_only && setup_automatic_update_checks; then
         echo # For visual separation
     fi
 
-    # Install the hooks into existing local repositories
-    if ! should_skip_install_into_existing_repositories; then
-        if is_single_repo_install; then
-            install_hooks_into_repo "$(pwd)" || return 1
-        else
+    if is_single_repo_install; then
```

Comment:
This seems to circumvent the `--skip-install-into-existing` for a) single repos, b) existing repos on the registered list.
I'd keep the old way for this flag, if we explicitly didn't want to install into existing repos, then just don't do any of these.